### ISSUE TITLE
Update all `secret: ` values in `.travis.yml`

### ISSUE THREAD

**Issue by webknjaz** [aio-libs/aioelasticsearch#103]
Do like this:
```
REPO_NAME={{ YOUR_REPO_NAME }}
travis encrypt -r "aio-libs/${REPO_NAME}" --api-endpoint 'https://api.travis-ci.com/'
```

Ref: https://github.com/orgs/aio-libs/teams/admins/discussions/9

**Comment by webknjaz** [comment]
user: aio-libs-bot
```yaml
secure: "cKvNQXCPwKAZHB02Me462NOyd4za1r5TJYUgCvrAkJTSf6qHdlLnOvH1bZtDyfs2GEpvYlx8nNZJHkbVHSiz3UEkaiuLj6xfaJdVUpJ9qK5w4YQvrMeb4cD6amIvMttrgMoppGq4GSjdcggBIwNcmgk2g5PrpozheK2GWkA1B8rlsnud4JRPhenzchH5yvv1VBXokVFvv6SyS+EIL8DEKRCZQ/Bug3N5QVPXWQn52JINY0c6v0UgLJzc0F82VCJKYdKkVBXLFGakwHlufjcn5TB9myia5hBeObdyXlJGk+NiGrZGchluAfE+QWfdtTn9AAyUHdNV7jjQWQtfv5gj025Zo2IXNRvHkeIG6CiHyZ3YiXorA20kZgaHd95PG+WkJwbRUBJdCgrZ1r+saxrA8D7tT3aljEwc10jz/YcQdazQmFXqV2NP0PDAiqNS0zgIPPZQn2lo+KUBpiUA+h/Wp/Jv3XlmhyyvAeCfGVMDdBRhIq5SDHdaqVJ+UbGR79qKcA9/bKYimyZ83MRUBUsZPMh/VElLKImN3LqjCjsoEeFCeh5pZvEmrydNRv4aeE124nnYX0DY6Md6+NN6wjnBagg+Ws4cR0UzsQrO35L/JECEEklHQv+nGn4sTmMoHwf9IR0XeNwol3dnY/3HVuipgNcFN84Ir7F4gcUtshJu0Vo="
```

**Comment by webknjaz** [comment]
Resolved by #161
